### DISCUSSION TITLE
How do I change Icon theme?

### DISCUSSION THREAD

**Discussion by SarjyantShrestha** [prasanthrangan/hyprdots#1375] [Q&A] [answered]
I manually installed only one (Rose Pine) hyde theme.
I tried to change icon theme using gtk settings, from Tela circle pink to tela circle dracula and it's not working.
I even tried
`exec = gsettings set org.gnome.desktop.interface icon-theme 'Tela-circle-dracula'`
on ~/.config/hypr/themes/theme.conf and ~/.config/hyde/themes/Rosé\ Pine/hypr.theme
and it doesn't seem to work.

**Comment by dieBakterie** [comment]
Try it with nwg-look

**Reply by SarjyantShrestha** [reply]
gtk settings / nwg-look same thing no?

**Reply by dieBakterie** [reply]
Nwg-look is a wlroots optimized Version of gtksettings

**Reply by SarjyantShrestha** [reply]
its the same thing

**Reply by dieBakterie** [reply]
Nwg-look is an wlroots wayland optimized Version of gtksettings

**Reply by SarjyantShrestha** [reply]
someone help! I still haven't figured it out.

**Comment by kRHYME7** [comment, marked as answer]
> I even tried
exec = gsettings set org.gnome.desktop.interface icon-theme 'Tela-circle-dracula'
on ~/.config/hypr/themes/theme.conf and ~/.config/hyde/themes/Rosé\ Pine/hypr.theme
and it doesn't seem to work.

> ~/.config/hyde/themes/Rosé\ Pine/hypr.theme

This is the right way, but you need to at least change wallpapers or themes after changing those lines. 
 Be sure that the icon-theme is available too. 


I suggest copy a theme as template and build your own theme and rename it to something. 
It is simple as renaming the folder of the theme.

**Reply by SarjyantShrestha** [reply]
Thank you! it worked.

**Reply by rackedupkitty** [reply]
Hey, im having a similar issue here, the icon theme is working excellently except for on dolphin, everything else seems to work though, do you have any ideas why this may be, and how I could go about fixing this?

**Reply by kRHYME7** [reply]
check the latest changes

**Reply by rackedupkitty** [reply]
Its been days, I finally figured it out after the comments on the bug report I put on the GitHub. Thank you so much for all of your help, in hindsight I feel like a complete moron for asking about this. For anyone else who has the same issue:
1. nano ~/.config/kdeglobals
2. add the following:
[Icons]
theme=your-icon-theme
